libraries/Microsoft.Bot.Builder.Dialogs.Adaptive/Schemas/Actions/Microsoft.Foreach.schema

```diff
@@ -19,7 +19,7 @@
                 "$type": "Microsoft.IDialog"
             },
             "title": "Actions",
-            "description": "Actions to execute"
+            "description": "Actions to execute for each item."
```

Comment:
Actions to execute for each item.  Use $foreach.value to get access to the value of each item.

Reply:
Both each and for each page appear in the menu - the word 'item' allows us to clearly differentiate one from the other while keeping the rest of the sentence/description parallel.